oap-server/server-starter/src/main/resources/lal/default.yaml

```diff
@@ -16,6 +16,7 @@
 # The default LAL script to save all logs, behaving like the versions before 8.5.0.
 rules:
   - name: default
+    layer: UNDEFINED
```

Comment:
This should be general service layer

Reply:
I feel we may not need a default LAL file. You could change this LAL for general service only.